useragent/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```

Comment:
Yep. Just updated the README to say so too. Old packages should also be gradually updated to support this.